### DISCUSSION TITLE
Contents shown in the "Follow" drop down menu

### DISCUSSION THREAD

**Discussion by ulbrichdennis** [academicpages/academicpages.github.io#1971] [Q&A] [answered]
On smaller screens, there is this little drop down menu called "Follow" where a lot of things are linked or shown without links. Where can I change which things appear there and where can I add links?
For example, I do not like the location to be shown there for two reasons: First, it has no link and second it is nothing to follow in my opinion. So I would either like to add a link to the location (to make it something to "follow") or to delete it from the drop down menu.

**Comment by rjzupkoii** [comment, marked as answer]
> Where can I change which things appear there and where can I add links?

If it's social media then it's controlled based upon if you a profile listed as part of `_config.yml` with the default behavior to render everything in that sidebar. However, Issue #1970 has been created and the plan will be to add a bit of control over what doesn't get rendered in mobile views.
